crates/store/src/genesis/config/samples/01-simple.toml

```diff
@@ -1,6 +1,11 @@
 timestamp = 1717344256
 version   = 1
 
+
+[fee_parameters]
+symbol                = "MIDEN"
+verification_base_fee = 0
+
 [[fungible_faucet]]
 decimals     = 3
 max_supply   = 100_000_000
```

Comment:
I'm assuming we left the new approach for defining these (i.e., something like in https://github.com/0xMiden/miden-node/pull/1143#discussion_r2265981740) for the future - right? Let's create an issue for refactoring this later.

Reply:
I prefer simplicity until we have the means to make an educated choice of how it's going to be used, and didn't want to block the client any longer on that discussion specifically